rskj-core/src/main/java/co/rsk/peg/bitcoin/P2shP2wshCustomErpRedeemScriptBuilder.java

```diff
@@ -0,0 +1,55 @@
+package co.rsk.peg.bitcoin;
+
+import co.rsk.bitcoinj.core.BtcECKey;
+import co.rsk.bitcoinj.core.Utils;
+import co.rsk.bitcoinj.script.Script;
+import co.rsk.bitcoinj.script.ScriptBuilder;
+import co.rsk.bitcoinj.script.ScriptOpCodes;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+
+public class P2shP2wshCustomErpRedeemScriptBuilder implements ErpRedeemScriptBuilder {
+    private static final Logger logger = LoggerFactory.getLogger(P2shP2wshCustomErpRedeemScriptBuilder.class);
+
+    private P2shP2wshCustomErpRedeemScriptBuilder() {}
+
+    public static P2shP2wshCustomErpRedeemScriptBuilder builder() {
+        return new P2shP2wshCustomErpRedeemScriptBuilder();
+    }
+
+    @Override
+    public Script of(
+        List<BtcECKey> defaultPublicKeys,
+        int defaultThreshold,
+        List<BtcECKey> emergencyPublicKeys,
+        int emergencyThreshold,
+        long csvValue
+    ) {
+        logger.debug("[of] Creating the redeem script from the scripts");
+
+        Script customRedeemScript = ScriptBuilder.createCustomRedeemScript(defaultThreshold, defaultPublicKeys);
+
+        // no need for using the new redeem script format since the standard Multisig one supports up to 15 keys
```

Comment:
```suggestion
        // no need for using the new redeem script format for the emergency keys since the ¿normal? ¿common? m/n Multisig one supports up to 15 keys
```
noticing now, i'll avoid the word `standard multisig` since we use that for referencing a federation without erp keys

Reply:
no need for using the new redeem script format for the emergency keys since the regular Multisig supports up to 15 keys

Reply:
nice, go ahead